### ISSUE TITLE
Ignore step cause report cannot group example

### ISSUE THREAD

**Issue by katsucpe** [serenity-bdd/serenity-jbehave#15]
when in scenario have !-- for ignore some step or just for do a comment
the report cannot group steps to each example. I have to remove ignore for make it work.

Do I need to config something?
 
![2015_07_29_13_04_26_04 1_validate_if_the_user_uploads_the_data_to_create_table_by_using_cdi_self_ser](https://cloud.githubusercontent.com/assets/8433882/8951005/01441130-35f3-11e5-859c-9c6f8bb42e49.png)

**Comment by YamStranger** [comment]
Hello @katsucpe 

It seems that this bug was fixed during moving from jbehave 3.0.x to jbehave 4.0.5 and can be tested in next release by you. 

I created some example to check how your code work now: 

So, I created simple test: 

```
Narrative:
In order check reports
As an Developer
I want to workin with todos

Meta:
@tag component:UI
@tag tool:jbehave
Scenario: Record a some todo actions for future use
Given I need to buy some milk
!-- When I add the todo action "Buy some milk"
!-- And I add the todo action "Buy some tea"
!-- Then "Buy some milk" should be recorded in my todo list

Scenario: All todos should be marked as Active
Given I need to buy some milk
When I add the todo action "Buy some milk"
And I add the todo action "Buy some tea"
Then "Buy some milk" should be recorded in the "Active" items
!-- Then "Buy some tea" should be recorded in the "Active" items
```

Here is reports for it: 
![commented_story](https://cloud.githubusercontent.com/assets/1667699/13280809/cfb341fe-dae8-11e5-8784-2c5950dfedc8.png)
![comments_in_scenario](https://cloud.githubusercontent.com/assets/1667699/13280810/cfb6b2b2-dae8-11e5-8fe9-14aac9b80735.png)
![comments_in_scenario_2](https://cloud.githubusercontent.com/assets/1667699/13280811/cfb9e3a6-dae8-11e5-8885-28a82eac4256.png)

If you find fix that you expected and your issue already solved please close this ticket. 

Thanks a lot!

**Comment by YamStranger** [comment]
It seems that this ticket already fixed, and ticket can be closed.
